tap_eloqua/schema.py

```diff
@@ -151,6 +153,8 @@ def get_bulk_schema(client,
 
         if prop in pk:
             inclusion = 'automatic'
+        elif prop in SYSTEM_FIELDS:
+            inclusion = 'automatic'
```

Comment:
This should probably use `set(system_fields)` (the parameter to the function) instead of a global set. Activity (and maybe others?) exports have different combinations of system fields.